### ISSUE TITLE
Segfault on Anonymous Class with a (not yet started) Worker

### ISSUE THREAD

**Issue by tpunt** [krakjoe/pthreads#658]
Environment
==========

  * PHP: PHP 7.1.0 (cli) ( ZTS DEBUG )
  * pthreads: 3.1.7dev
  * OS: Mac

Reproducing Code
==============

```php
$worker = new Worker();
$task = new class extends Threaded {
    public static $a;
};
$task::$a = new class extends Threaded {};
$worker->start();
$worker->stack($task);
$worker->shutdown();
```

The worker must not be started before the static property assignment. `Pool`s are also affected:

```php
$task = new class extends Threaded {
    public static $a;
};
$task::$a = new class extends Threaded {};
$pool = new Pool(2);
$pool->submit($task);
$pool->shutdown();
```

**Comment by krakjoe** [comment]
I had a go at #658 and #659 this morning ... but failed ... I'll have to come back to it ...

**Comment by tpunt** [comment]
Resolved by #686.
